ev3dev2/motor.py

```diff
@@ -1781,13 +1863,47 @@ def on_for_degrees(self, left_speed, right_speed, degrees, brake=True, block=Tru
         self.right_motor._set_rel_position_degrees_and_speed_sp(right_degrees, right_speed_native_units)
         self.right_motor._set_brake(brake)
 
+        left_motor_runtime_eta = float(left_degrees / left_speed_native_units)
+        right_motor_runtime_eta = float(right_degrees / right_speed_native_units)
+        left_motor_position_init = self.left_motor.position
+        right_motor_position_init = self.right_motor.position
+
+        log.debug("{}: on_for_degrees {}".format(self, degrees))
+        log.debug("{}: left_speed {}, left_speed_native_units {}, left_degrees {}, left-position {}->{}, ETA {}s".format(
```

Comment:
Do we have any idea if debug logging has a perf impact?

Reply:
I have tried to measure its impact in the past but the delta was so small I couldn’t really measure it. Unless you do something crazy like drop into a loop and call log.debug 1000x I don’t think you will ever notice an impact.

Reply:
On the EV3? It sounds believable either way, so I'm fine.

Reply:
yep I tested on EV3 back when I looked into it